src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nr25-01.mdx

```diff
@@ -0,0 +1,190 @@
+
+   Additionally, New Relic recommends that all customers identify any other uses of Fluent Bit in their environments and update them to at least version 3.2.7.
+
+3. **Once I update to the latest versions of the listed log forwarding services, do I have to do anything else?**
+
+  Yes, but only if you are running Infrastracture agents on Linux hosts. If you are running the Linux Infrastructure agent, you will also need to [update Fluent Bit](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/#install-fb-version) within your environment to a version 3.2.7 or later.
+
+4. **How can I find out which Fluent Bit version I’m using?**
+
+  If you previously used New Relic log forwarding instrumentation (listed above), and used the affected plugins, you might still be impacted. New Relic recommends that you upgrade your agents, or at minimum disable the affected plugins.
+
+  Additionally, New Relic recommends that all customers identify any other uses of Fluent Bit in their environments and update them to at least version 3.2.7.
+
+<table>
+  <thead>
+    <th>
+      <td>
+        Agent
+      </td>
+    </th>
+
+    <th>
+      <td>
+        Steps
+      </td>
+    </th>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        Infra agent and standalone Fluent bit
+      </td>
+
+      <td>
+        * For your Infrastructure agents, navigate to the [Infrastructure Inventory](/docs/infrastructure/infrastructure-data/infrastructure-ui-pages/infra-inventory-ui-page/) UI and search for Fluent Bit.
+        * Run the following NRQL query:
+          `FROM Log select count(*) where (plugin.type like '%fluent%' or plugin.source like '%fluent%' or plugin.source like '%-fb-%') facet plugin.type, plugin.source, plugin.version`
+
+          Then, check which Fluent Bit version was installed with the [output plugin](https://github.com/newrelic/helm-charts/releases/tag/nri-bundle-5.0.115).
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Kubernetes
+      </td>
+
+      <td>
+        * For New Relic Logging Helm chart version 1.25.0 or higher, Navigate to the Installed tab and search for Fluent bit in the entities field
+
+        * Otherwise, run the following NRQL query: 
+          `FROM K8sContainerSample select latest(containerImage) Where podName like '%newrelic-logging%' FACET clusterName`
+
+          Then, check which Fluent Bit version was installed with the [output plugin](https://github.com/newrelic/helm-charts/releases/tag/nri-bundle-5.0.115).
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Supporting Release Notes
+
+[Fluent Bit release notes](/docs/release-notes/fluentbit-release-notes/)
+
+[Infrastructure Release Notes](/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/)
+
+[Kubernetes Integration Release Notes](/docs/release-notes/infrastructure-release-notes/kubernetes-integration-release-notes/3/)
```

Comment:
@gmanandhar-nr, we can strike this link to K8s release notes as there will be no changes to reference.